draft-ietf-rtcweb-jsep.xml

```diff
@@ -741,66 +741,80 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
+          or if the track is replaced with a different track.
+          </t>
+
+          <t>Depending on how the RtpSender is configured, it may be producing
+          a single resolution, or, if simulcast <xref target="sec.simulcast" />
+          has been negotiated, multiple specific resolutions. In addition,
+          depending on the configuration, each encoding may have the
+          flexibility to reduce resolution when needed, or may be locked to
+          a specific output resolution.
+          </t>
+
+          <t>For each encoding being produced by the RtpSender, the following
+          rules are applied to determine what should be transmitted:
+            <list style="symbols">
+              <t>First, the right "a=imageattr recv" attribute is selected.
+              While JSEP endpoints will not include more than one such
+              attribute per media format, non-JSEP endpoints may send
+              multiple relevant "a=imageattr recv" attributes in a m=
+              section. The correct attribute is selected by taking the
+              attribute with the highest "q=" value from the set of attributes
+              that reference the media format that has been selected for the
+              specified encoding. If multiple attributes have the same "q="
+              value, the one that appears first in the m= section is used.
+              </t>
+
+              <t>If there is an applicable "a=imageattr recv" attribute for
+              the encoding, the limits from the attribute are then compared to
+              the encoder resolution. Only the specific limits mentioned below
+              are considered; any other values, e.g. picture aspect ratio,
+              MUST be ignored.
+              Note that when considering a MediaStreamTrack that is producing
+              rotated video, the unrotated resolution MUST be used for the
+              checks. This is
+              required regardless of whether the receiver supports
+              performing receive-side rotation (e.g., through CVO
+              <xref target="TS26.114" />), as it significantly simplifies
+              the matching logic.</t>
+
+              <t>If the attribute includes a "sar=" (sample aspect ratio)
+              value set to something other than "1.0", indicating the
+              receiver wants to receive non-square pixels, this cannot be
+              satisfied and the sender MUST NOT transmit the encoding.</t>
+
+              <t>If the encoder resolution exceeds the maximum size permitted by the
+              attribute, and the encoder is allowed to adjust its resolution,
+              the encoder SHOULD apply downscaling in order to satisfy the limits,
+              although the downscaling MUST NOT change the track aspect ratio.
+              If downscaling cannot be applied, the encoding MUST NOT be
+              transmitted, and an error SHOULD be surfaced to the application.</t>
```

Comment:
Is this saying that I must pan-and-scan rather than doing pillarboxing or letterboxing?